File: dmn-core/src/test/resources/tck/cl3/expected/java/dmn/0004-lending/PreBureauRiskCategoryTableRuleOutput.java

import java.util.*;

@javax.annotation.Generated(value = {"decisionTableRuleOutput.ftl", "PreBureauRiskCategoryTable"})
public class PreBureauRiskCategoryTableRuleOutput extends com.gs.dmn.runtime.RuleOutput {
    private String preBureauRiskCategoryTable;

    public PreBureauRiskCategoryTableRuleOutput(boolean matched) {
        super(matched);
    }

    public String getPreBureauRiskCategoryTable() {
        return this.preBureauRiskCategoryTable;
    }
    public void setPreBureauRiskCategoryTable(String preBureauRiskCategoryTable) {
        this.preBureauRiskCategoryTable = preBureauRiskCategoryTable;
    }

    @Override
    public boolean equals(Object o) {
        if (this == o) return true;
        if (o == null || getClass() != o.getClass()) return false;

        PreBureauRiskCategoryTableRuleOutput other = (PreBureauRiskCategoryTableRuleOutput) o;
        if (this.getPreBureauRiskCategoryTable() != null ? !this.getPreBureauRiskCategoryTable().equals(other.getPreBureauRiskCategoryTable()) : other.getPreBureauRiskCategoryTable() != null) return false;

        return true;
    }

    @Override
    public int hashCode() {
        int result = 0;
        result = 31 * result + (this.getPreBureauRiskCategoryTable() != null ? this.getPreBureauRiskCategoryTable().hashCode() : 0);

        return result;
    }

    public String toString() {
        StringBuilder result_ = new StringBuilder("(matched=" + isMatched());
        result_.append(String.format(", preBureauRiskCategoryTable='%s'", preBureauRiskCategoryTable));
        result_.append(")");
        return result_.toString();
    }
}

File: dmn-core/src/test/resources/tck/proto/expected/proto3/kotlin/dmn/0004-lending/PreBureauRiskCategoryTableRuleOutput.kt

@javax.annotation.Generated(value = ["decisionTableRuleOutput.ftl", "PreBureauRiskCategoryTable"])
class PreBureauRiskCategoryTableRuleOutput(matched: Boolean) : com.gs.dmn.runtime.RuleOutput(matched) {
    @com.fasterxml.jackson.annotation.JsonProperty("PreBureauRiskCategoryTable")
    var preBureauRiskCategoryTable: String? = null

    override fun equals(o: Any?): Boolean {
        if (this === o) return true
        if (javaClass != o?.javaClass) return false

        val other = o as PreBureauRiskCategoryTableRuleOutput
        if (if (this.preBureauRiskCategoryTable != null) this.preBureauRiskCategoryTable != other.preBureauRiskCategoryTable else other.preBureauRiskCategoryTable != null) return false

        return true
    }

    override fun hashCode(): Int {
        var result = 0
        result = 31 * result + (if (this.preBureauRiskCategoryTable != null) this.preBureauRiskCategoryTable.hashCode() else 0)

        return result
    }

    override fun toString(): String {
        val result_ = StringBuilder("(matched=" + isMatched())
        result_.append(String.format(", preBureauRiskCategoryTable='%s'", preBureauRiskCategoryTable))
        result_.append(")")
        return result_.toString()
    }
}
